charts/prefect-server/README.md

```diff
@@ -24,6 +24,22 @@ kubectl port-forward svc/prefect-server 4200:4200
 
 Note: If you choose to make modifications to either the `server.prefectApiUrl` or `service.port`, make sure to update the other value with the updated port!
 
+## Loop Services Configuration
+
+The Prefect server includes background loop services related to scheduling and cleanup. By default, these run in the same deployment as the web server, but they can be separated for better resource management and scalability.
+
+To run loop services in a separate deployment:
+
+```yaml
+loopServices:
```

Comment:
i know it's not technically the nomenclature we use, but what if we call this `backgroundServices`? do people know what a loop service is?

Reply:
more of a q for @zzstoatzz

Reply:
LoopService is internal nomenclature so I'm totally cool with `backgroundServices` instead

wdyt @cicdw

Reply:
im good with backgroundServices - updating now